build/azure-pipelines/darwin/sql-product-build-darwin.yml

```diff
@@ -107,13 +113,26 @@ steps:
     displayName: Run unit tests
     condition: and(succeeded(), eq(variables['RUN_TESTS'], 'true'))
 
+  - script: |
+      set -e
+      pushd ../azuredatastudio-darwin
+      codesign --force --entitlements $(Build.SourcesDirectory)/build/azure-pipelines/entitlements.xml -s 'sqlkey' *.app
+      popd
+    displayName: 'Manual codesign'
+
   - script: |
       set -e
       mkdir -p .build/darwin/archive
       pushd ../azuredatastudio-darwin && zip -r -X -y $(Build.SourcesDirectory)/.build/darwin/archive/azuredatastudio-darwin.zip * && popd
     displayName: 'Archive'
 
-  - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@1
+  - task: PublishPipelineArtifact@0
```

Comment:
Publishing and signing should be done in the createDrop.sh file for flexibility.

Reply:
These publish steps are only here temporarily

Reply:
Anyway we can't put this signing step in there, since we still need to sign with ESRP afterwards

Reply:
Why are we still publishing this artifact?

Reply:
I don't trust that this notarization is going to be super stable yet, so I'm keeping it around for another month or so to make sure it doesn't cause any issues. I want to be able to get any specific binary in these steps.

Reply:
Is that being tracked somewhere? its easy to forget things that are a month out if no one is actually being impacted by it.

Reply:
Yes - I have it in my list of TODOs.

I'll add it as an issue as well.